### PR TITLE
docs: don't recommend elasticsearch

### PR DESCRIPTION
**Pull request by igor-hpe**
## Description

docs on elasticsearch:
- provide example config
- no longer recommend it

## Test Plan

N/A

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

No Ticket

### CODE REVIEW

File: docs/setup-cluster/elasticsearch-logging-backend.rst [hunk 1]
```diff
@@ -4,24 +4,25 @@
...
-is likely to be limited to a log throughput of around hundreds of megabytes per second; we recommend
-moving to Elasticsearch to get past that limit.
+.. code:: yaml
+
+   logging:
+   type: elastic
+   host: "elastic.example.com"
+   port: 443
+   security:
+      username: "elastic-user"
```
Comment (dzhu) [inline docs/setup-cluster/elasticsearch-logging-backend.rst:19]: Use two-space indentation.

File: docs/setup-cluster/elasticsearch-logging-backend.rst [hunk 2]
```diff
@@ -4,24 +4,25 @@
...
+      password: "mypassword"
+      tls:
+      enabled: true
```
Comment (dzhu) [inline docs/setup-cluster/elasticsearch-logging-backend.rst:22]: Indent.

File: docs/setup-cluster/elasticsearch-logging-backend.rst [hunk 3]
```diff
@@ -64,5 +65,3 @@ on how the cluster will be used:
      }
    }
```
Comment (dzhu) [inline docs/setup-cluster/elasticsearch-logging-backend.rst:67]: Remove blank line. (You can automatically apply the correct formatting by running `make -C docs fmt`, or set up your editor to apply `rstfmt` to reStructuredText files on save.)

File: docs/setup-cluster/elasticsearch-logging-backend.rst [hunk 4]
```diff
@@ -4,24 +4,25 @@
...
-database's CPU hits ~90%). To get a little more mileage from the default, we recommend increasing
-the capacity of the database. At a certain point, the master instance itself will become the
-bottleneck, since it has limited incoming network bandwidth for HTTP requests delivering logs and
-limited resources to process them. The master instance size can be increased, but vertical scaling
-is likely to be limited to a log throughput of around hundreds of megabytes per second; we recommend
-moving to Elasticsearch to get past that limit.
+.. code:: yaml
+
+   logging:
+   type: elastic
```
Comment (dzhu) [inline docs/setup-cluster/elasticsearch-logging-backend.rst:15]: Indent (including everything below).

File: docs/setup-cluster/elasticsearch-logging-backend.rst [hunk 5]
```diff
@@ -4,24 +4,25 @@
  Elasticsearch-backed logging
 ##############################
 
-This guide covers the limitations of the default logging backend, as a guideline on when to migrate
-to Elasticsearch, and some tips for how to tune Elasticsearch to work best with Determined.
+It is possible to use Elasticsearch as an alternative to the default logging backend. In the past,
+such configuration was recommended for larger installations, but it's no longer the case.
```
Comment (dzhu) [inline docs/setup-cluster/elasticsearch-logging-backend.rst:8]:
```suggestion
such a configuration was recommended for larger installations, but that is no longer the case.
```

File: docs/setup-cluster/elasticsearch-logging-backend.rst (hunk 1 above)
Comment (igor-hpe) [reply docs/setup-cluster/elasticsearch-logging-backend.rst:19]: Thanks. I used the web editor before, but it seems that restructured is too picky. I'll edit again using VS Code with a linter installed.

File: docs/setup-cluster/elasticsearch-logging-backend.rst [hunk 6]
```diff
@@ -4,24 +4,22 @@
  Elasticsearch-backed logging
 ##############################
 
-This guide covers the limitations of the default logging backend, as a guideline on when to migrate
-to Elasticsearch, and some tips for how to tune Elasticsearch to work best with Determined.
+It is possible to use Elasticsearch as an alternative to the default logging backend. In the past,
+such configuration was recommended for larger installations, but it is no longer the case.
```
Comment (tara-det-ai) [inline docs/setup-cluster/elasticsearch-logging-backend.rst:8]: It is possible to use Elasticsearch as an alternative to the default logging backend.

(remove the second sentence)

**Review by tara-det-ai — COMMENTED** [review]
suggested [edit](https://github.com/determined-ai/determined/pull/7817/files#r1319178649)

File: docs/setup-cluster/elasticsearch-logging-backend.rst (hunk 5 above)
Comment (tara-det-ai) [reply docs/setup-cluster/elasticsearch-logging-backend.rst]: We don't need the second sentence, it's ambiguous and users won't understand the reference to time (when in the past did we recommend this?).

**Review by tara-det-ai — APPROVED** [review]
The content LGTM (I did not check the code).

**Pull request closed without merging**